dts/arm/st/wba/stm32wba.dtsi

```diff
@@ -156,6 +156,66 @@
 			};
 		};
 
+		pwr: power@46020800 {
+			compatible = "st,stm32-pwr";
+			reg = <0x46020800 0x400>;
+			status = "disabled";
```

Comment:
A recommendation for DTS coding style is to have the `status` property last in the node properties.
For consistency with the other nodes implementation, could you move it down?

Reply:
It's not very consistent regarding the other nodes in this DTSI file (aside the adc node) but I must admit it's consistent with the `pwr` nodes implementation in the other STM32 SoCs DTSI files.